olmo/config.py

```diff
@@ -1038,11 +1090,21 @@ class TrainConfig(BaseConfig):
     Settings for compiling the model with ``torch.compile()``.
     """
 
+    distributed_strategy: Optional[DistributedStrategy] = None
+    """
+    Distributed strategy for OLMo model (eg. single GPU, DDP, FSDP).
+    """
+
     fsdp: FSDPConfig = field(default_factory=FSDPConfig)
     """
     Fully sharded data parallel settings.
     """
 
+    ddp: DDPConfig = field(default_factory=DDPConfig)
+    """
+    DDP settings.
+    """
```

Comment:
I would make these optional now, otherwise we're going to look at some configs later and be like "why are there DDP settings in this config? I thought we ran this FSDP"
```suggestion
    fsdp: Optional[FSDPConfig] = field(default_factory=FSDPConfig)
    """
    Fully sharded data parallel settings.
    """

    ddp: Optional[DDPConfig] = None
    """
    DDP settings.
    """
```